core/src/main/java/com/scalar/db/storage/cassandra/SelectStatementHandler.java

```diff
@@ -1,5 +1,6 @@
 package com.scalar.db.storage.cassandra;
```

Comment:
It looks like we also need to quote column name for ordering?
https://github.com/scalar-labs/scalardb/blob/c6dc9766f2cc55cc92d8cb039a4c50fd42c1897f/core/src/main/java/com/scalar/db/storage/cassandra/SelectStatementHandler.java#L264
https://github.com/scalar-labs/scalardb/blob/c6dc9766f2cc55cc92d8cb039a4c50fd42c1897f/core/src/main/java/com/scalar/db/storage/cassandra/SelectStatementHandler.java#L266
https://github.com/scalar-labs/scalardb/blob/c6dc9766f2cc55cc92d8cb039a4c50fd42c1897f/core/src/main/java/com/scalar/db/storage/cassandra/SelectStatementHandler.java#L269

If yes, please also add tests for it.

Reply:
We need it as well, the scan test with the range also includes the order in the query but seems the `two` keyword I used does not affect the query in this case. I change the keyword to `select` then the test failed. I added the quote for ordering and changed the column name as well [commit](https://github.com/scalar-labs/scalardb/pull/441/commits/2d136ee2f33c57a51b6b960ad43363529968faa7)

Reply:
So you mean the `two` keyword is not an actual reserved keyword, right? If so, please check if all the reserved keywords you use in the integration tests are really reserved keywords?

Reply:
In the DataStax document, the `two` is mentioned in the reserved keyword list but seems it depends on where the keyword is placed in the query, some still working. For some obvious keywords like `select`, `update` should be really a keyword

Reply:
Okay I see. It would be great if you re-check the reserve keywords for all the adapters just in case.